core/src/main/java/io/github/wysohn/triggerreactor/core/manager/trigger/inventory/InventoryTrigger.java

```diff
@@ -82,6 +82,11 @@ public IItemStack[] getItems() {
         return items;
     }
 
+    public boolean isPickup() {
```

Comment:
This is good too, but I think it'd be better to `#canPickup()`.

Reply:
Oh i think so